mesheryctl/internal/cli/root/orgs/org.go

```diff
@@ -0,0 +1,36 @@
+package orgs
+
+import (
+	"fmt"
+
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var (
+	availableSubcommands = []*cobra.Command{ListOrgCmd}
+)
+var OrgCmd = &cobra.Command{
+
+	Use:   "org",
+	Short: "view list of registered orgs",
```

Comment:
```suggestion
	Short: "Details registered organizations",
```
Also we need a Long description as wel

Reply:
Please mark this as complete if implemented